Jenkinsfile

```diff
@@ -311,7 +311,8 @@ def Tasks = [
   "tools": '''
     setJavaVersion $java
     npm install &&
-    sbt ++$scala ir$v/test logging$v/compile linkerInterface$v/compile \
+    sbt ++$scala ir$v/test logging$v/compile \
+        linkerInterface$v/test linkerInterfaceJS$v/test \
```

Comment:
This should not be here. But in the `bootstrap` task. (the `testInterfaceJS$v/compile` was implicit in the mima check there, sorry; didn't realize that).

Reply:
Ah, I see, all JS tasks go in `bootstrap` and JVM tasks in `tools`. Fixed in 0c9e543c35dc32a34681040579c1afd36fa18e9b